pkg/3scale/amp/component/highavailability_options.go

```diff
@@ -3,20 +3,25 @@ package component
 import "github.com/go-playground/validator/v10"
 
 type HighAvailabilityOptions struct {
-	AppLabel                            string `validate:"required"`
 	BackendRedisQueuesEndpoint          string `validate:"required"`
-	BackendRedisQueuesSentinelHosts     string `validate:"required"`
-	BackendRedisQueuesSentinelRole      string `validate:"required"`
-	BackendRedisStorageEndpoint         string `validate:"required"`
-	BackendRedisStorageSentinelHosts    string `validate:"required"`
-	BackendRedisStorageSentinelRole     string `validate:"required"`
+	BackendRedisQueuesSentinelHosts     string
+	BackendRedisQueuesSentinelRole      string
+	BackendRedisStorageEndpoint         string
```

Comment:
I think `BackendRedisStorageEndpoint` might be better to be required because you always have to provide it